apps/lfx-one/src/app/modules/dashboards/executive-director/components/email-ctr-drawer/email-ctr-drawer.component.html

```
@@ -0,0 +1,144 @@
+  <div class="flex flex-col gap-6 pb-2" data-testid="email-ctr-drawer-content">
+    <!-- Summary Stats -->
+    <div class="flex gap-4" data-testid="email-ctr-drawer-stats">
+      <div class="flex-1 flex flex-col gap-1 p-4 border border-gray-200 rounded-lg">
+        <span class="text-sm text-gray-500">Current CTR</span>
+        <span class="text-2xl font-semibold text-gray-900">{{ data().currentCtr.toFixed(2) }}%</span>
+      </div>
+      <div class="flex-1 flex flex-col gap-1 p-4 border border-gray-200 rounded-lg">
+        <span class="text-sm text-gray-500">Month-over-Month</span>
+        <div class="flex items-center gap-2">
+          <span class="text-2xl font-semibold" [class]="data().trend === 'up' ? 'text-green-600' : 'text-red-600'">
+            {{ data().changePercentage > 0 ? '+' : '' }}{{ data().changePercentage }}%
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)